.travis.yml

```diff
@@ -54,3 +54,13 @@ jobs:
         - go get github.com/mattn/goveralls
       after_success:
         - $GOPATH/bin/goveralls -coverprofile=profile.cov -service=travis-ci
+    - stage: 'Integration tests'
+      env: GIMME_GO_VERSION=1.12.x FSC_PATH="/tmp/fsc-repo" TAGS=@FEATURE_ROLLOUT && ~@INPUT_FILTER"
+      before_script:
+        - mkdir -p $FSC_PATH
+        - pushd $FSC_PATH && git init && git fetch --depth=1 https://$CI_USER_TOKEN@github.com/optimizely/fullstack-sdk-compatibility-suite ${FSC_BRANCH:-master} && git checkout FETCH_HEAD && popd        
+      install:
+        - eval "$(gimme)"
+      script:
+        - ./scripts/run-fsc-tests.sh -f $FSC_PATH/features -d $FSC_PATH/features/support/datafiles/ -t "$TAGS"        
```

Comment:
@yasirfolio3 please let me know which tags to add in TravisJob env variable. Is the key going to be just `TAGS`? I can add it tonight so you can continue your work

Reply:
Hey @mikeng13 , i have added `TAGS` as env in travis settings, only 1 test is failing with the added tags which is due to activate listener.